### PR TITLE
Scenarios: soft delete and temporary display of "soft deleted" scenarios

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

File: client/routes/index.jsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const NewScenario = props => {
 
 const Navigation = () => {
     return (
-        <Menu className="nav">
+        <Menu className="nav" stackable>
```
Comment (ktbee) [inline client/routes/index.jsx]: I don't think we want this change in this PR?

Comment (rwaldron) [reply client/routes/index.jsx]: No, that was a mistake. I will remove

**Comment by rwaldron** [comment]
> I just had one question inline and another thing to confirm: this PR doesn't cover the front end functionality for soft deleting scenarios? Like when I click the trash icon in the editor, it's expected that it doesn't soft delete yet?

Yes, it does. When you hit the trash icon on a scenario, then confirm your intention to delete, it "soft deletes" the scenario and sends you back to the main dashboard, where the scenario will now appear at the end, with a "strike through"

File: client/routes/index.jsx (hunk 1 above)
Comment (rwaldron) [reply client/routes/index.jsx]: Removed!

**Comment by ktbee** [comment]
Follow up task cc @rwaldron https://app.asana.com/0/1127815256483386/1150846187283483

**Pull request merged**